openedx/features/course_bookmarks/templates/course_bookmarks/course-bookmarks.html

```diff
@@ -0,0 +1,58 @@
+## mako
+
+<%! main_css = "style-main-v2" %>
+
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%def name="online_help_token()"><% return "courseware" %></%def>
+<%def name="course_name()">
+<% return _("{course_number} Courseware").format(course_number=course.display_number_with_default) %>
+</%def>
+
+<%!
+import json
+from django.utils.translation import ugettext as _
+from django.template.defaultfilters import escapejs
+
+from django_comment_client.permissions import has_permission
+from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
+from openedx.core.djangolib.markup import HTML
+%>
+
+<%block name="bodyclass">course</%block>
+
+<%block name="pagetitle">${course_name()}</%block>
+
+<%include file="../courseware/course_navigation.html" args="active_page='courseware'" />
+
+<%block name="head_extra">
+${HTML(outline_fragment.head_html())}
```

Comment:
See other comments below where this template is used.   Is `outline_fragment` just a copy/paste issue, or is this have anything to do with the outline?